airflow/api_connexion/openapi/v1.yaml

```diff
@@ -4101,6 +4267,35 @@ components:
           description: The dataset event creation time
           nullable: false
 
+    DatasetDagRunQueue:
+      type: object
+      properties:
+        dataset_id:
```

Comment:
Should this show the uri instead? The if is not useful to end users.

Reply:
indeed, I'll address this one later

Reply:
Just addressed this one